docs/jwst/cube_build/main.rst

```diff
@@ -229,9 +229,15 @@ weighting
 
 The best algorithm with which to combine the irregularly-distributed samples of the point cloud to a rectilinear
 data cube is the subject of ongoing study, and depends on both the optical characteristics of the IFU and
-the science goals of a particular observing program.  At present, the default method uses a flux-conserving
+the science goals of a particular observing program.  At present there are two approaches to weighting the detector pixels.
+The default method uses a flux-conserving
 variant of Shephards method in which the value of a given element of the cube is a distance-weighted average
```

Comment:
minor nit, but "Shepards" should be possessive (i.e. Shepard's) or rephrase to "... variant of the Shepard method ..."